drivers/hid/hid-nintendo.c

```diff
@@ -1354,6 +1377,44 @@ static void joycon_parse_report(struct joycon_ctlr *ctlr,
 		input_report_key(dev, BTN_EAST, btns & JC_BTN_A);
 		input_report_key(dev, BTN_SOUTH, btns & JC_BTN_B);
 	}
+	if (jc_type_is_nescon(ctlr) || \
+	    jc_type_is_snescon(ctlr) || \
+	    jc_type_is_gencon(ctlr)) {
```

Comment:
I am undecided on if I should define a helper macro to the effect of `jc_type_is_nso` to include all the NSO controllers at once. That said, they share a few characteristics that would make such a helper macro reasonable (lack of motion and rumble, for example).

Happy to get input on this.